internal/flags.go

```diff
@@ -67,6 +68,7 @@ func parseFlags(args []string, stdout io.Writer) (*InputFlags, *ReturnError) {
 	flags.BoolVar(&inputFlags.summary, "summary", false, "display a summary of the changes instead of the full diff")
 	flags.BoolVar(&inputFlags.breakingOnly, "breaking-only", false, "display breaking changes only (old method)")
 	flags.BoolVar(&inputFlags.checkBreaking, "check-breaking", false, "check for breaking changes (new method)")
+	flags.BoolVar(&inputFlags.changelog, "changelog", false, "output to changelog format")
```

Comment:
suggest to rephrase with the word 'format' to avoid confusion with `-format` flag